services/processor/Dockerfile

```diff
@@ -1,9 +1,10 @@
 # In order for this Dockerfile to work it needs to be built from the `services` directory
-FROM python:alpine3.18
+FROM python:alpine3.16
 ENV PYTHONUNBUFFERED=1
+ENV CRYPTOGRAPHY_DONT_BUILD_RUST=1
 
 # Install python/pip
-RUN apk add --update --no-cache git curl build-base libffi-dev
+RUN apk add --update --no-cache git curl build-base libffi-dev py3-pip
```

Comment:
Why is this different from the other `Dockerfile` ?

Reply:
this is just a remnant of testing why the base `python:3.11-alpine3.18` doesn't work, I'll remove it.